2b_portfolio_prep_supervisor_workflow.R

```diff
@@ -12,7 +12,34 @@ prep_pf <- raw_pf %>%
 
 prep_pf <- add_meta_portfolio(prep_pf, inc_meta_portfolio)
 
-port_raw_all_eq <- prep_pf %>%
+# taken from add_portfolio_flags()
+prep_pf <- check_isin_format(prep_pf)
+prep_pf <- check_valid_value_usd(prep_pf)
+
+# taken from add_flags()
+prep_pf <- prep_pf %>%
+  mutate(
+    flag = case_when(
+      !has_valid_value_usd ~ "Negative or missing input value",
+      !has_valid_isin ~ "Invalid or missing ISIN",
+      TRUE ~ "Included in analysis"
+    )
+  )
+
+# taken from overall_validity_flag()
+portfolio_total <- prep_pf %>%
+  mutate(valid_input = case_when(
+    !has_valid_value_usd ~ FALSE,
+    !has_valid_isin ~ FALSE,
+    TRUE ~ TRUE
+  ))
```

Comment:
is this the same as
```r
mutate(valid_input = all(has_valid_value_usd, has_valid_isin))
```
?

Reply:
looks like it is. I stuck to the same style as in `add_portfolio_flags()`.. but I can change that, if preferred

Reply:
fine as this, just thought the syntax was bit over-complicated... if it reads well to you, leave it